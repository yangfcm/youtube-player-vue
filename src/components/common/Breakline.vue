<template>
  <div class="ui horizontal divider"></div>
</template>

<script>
export default {};
</script>

<style scoped></style>
