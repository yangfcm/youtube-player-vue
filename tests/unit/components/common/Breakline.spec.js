import { shallowMount } from "@vue/test-utils";
import Breakline from "@/components/common/Breakline";

test("Breakline component can render correctly", () => {
  const wrapper = shallowMount(Breakline);
  expect(wrapper).toMatchSnapshot();
});
