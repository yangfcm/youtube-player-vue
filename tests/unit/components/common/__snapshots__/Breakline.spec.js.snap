// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breakline component can render correctly 1`] = `<div class="ui divider"></div>`;
